tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -863,54 +857,6 @@ private function wipeDevice(\PluginFlyvemdmAgent $agent, $wipe = true, $expected
       $this->string($row['message'])->isEqualTo(json_encode($mqttMessage, JSON_UNESCAPED_SLASHES));
    }
 
-   /**
-    * Create a new invitation
-    *
-    * @param string $guestEmail
-    * @return \PluginFlyvemdmInvitation
-    */
-   private function createInvitation($guestEmail) {
```

Comment:
I don't see were do you move this method, this is necesary for the enrollment process

Reply:
the methid moved in  the test pachage (see the refactor branch, planned to merge into master if these tests pass)

Reply:
here : https://github.com/flyve-mdm/backend-tests/blob/refactor/src/Flyvemdm/Test/CommonTestCase.php#L42

Reply:
as discussed today, now all t e st helper mehods moved in  the folter tests/src.

Package backend-tests becomes obsolete